packages/mdc-drawer/_mixins.scss

```diff
@@ -147,3 +147,15 @@
   @include mdc-drawer-header_;
   @include mdc-drawer-list_;
 }
+
+@mixin mdc-drawer-fill-color($color) {
+  background-color: $color;
+}
+
+@mixin mdc-drawer-ink-color($color) {
+  color: $color;
+}
+
+@mixin mdc-drawer-scrim-color($color, $opacity) {
+  background-color: rgba($color, $opacity);
```

Comment:
You'll need to use the `mdc-theme-prop-value` function here to resolve special aliases like `primary` to their literal color values.

Since the `rgba` Sass function returns a literal color (and not an alias), you _don't_ need to use the `mdc-theme-prop` mixin here - just the similarly-named function to get the value.

Does that make sense?

Reply:
Think so...something like:

```
background-color: rgba(@include mdc-theme-prop-value($color), $opacity);
```

Reply:
Really close:

```scss
background-color: rgba(mdc-theme-prop-value($color), $opacity);
```

In Sass, `@include` is used when you call a `@mixin` like `mdc-theme-prop`.

A `@function` like `mdc-theme-prop-value`, on the other hand, doesn't get `@include`.

Reply:
fixed